src/js/components/projects/detail.tsx

```diff
@@ -258,6 +280,10 @@ const ProjectDetail = (props: RouteComponentProps) => {
     );
   }
 
+  // Redirect once project is deleted
+  if (!project && repositorySlug) {
+    return <Redirect to={routes.repository_detail(repositorySlug)} />;
+  }
```

Comment:
I assume this is what will happen, once the reducer filters the project that has just been deleted?